connectors/Planner/apiProperties.json

```diff
@@ -0,0 +1,410 @@
+          "x-ms-apimTemplate-policySection": "Request"
+        },
+        "templateId": "SetProperty",
+        "title": "Set odata type for assignment"
+      },
+      {
+        "parameters": {
+          "x-ms-apimTemplate-operationName": [
+            "CreateTask_V3",
+            "UnassignUsers",
+            "AssignUsers"
+          ],
+          "x-ms-apimTemplateParameter.propertyParentPath": "@body().assignments",
+          "x-ms-apimTemplateParameter.propertySubPath": "userId",
+          "x-ms-apimTemplateParameter.newPropertyPath": "@item().userId",
+          "x-ms-apimTemplate-policySection": "Request"
+        },
+        "templateId": "EncodePropertyValue",
+        "title": "Encode assignees"
+      },
+      {
+        "parameters": {
+          "x-ms-apimTemplate-operationName": [
+            "CreateTask_V3",
+            "UnassignUsers",
+            "AssignUsers"
+          ],
+          "x-ms-apimTemplateParameter.propertyParentPath": "@body().assignments",
+          "x-ms-apimTemplateParameter.propertySubPath": "userId",
+          "x-ms-apimTemplateParameter.sourceString": ".",
+          "x-ms-apimTemplateParameter.replacementString": "%2E",
+          "x-ms-apimTemplateParameter.newPropertyPath": "@item().userId",
+          "x-ms-apimTemplate-policySection": "Request"
+        },
+        "templateId": "StringReplace",
+        "title": "Encode dots in userId for OData"
+      },
+      {
+        "parameters": {
+          "x-ms-apimTemplate-operationName": [
+            "CreateTask_V3",
+            "UnassignUsers",
+            "AssignUsers"
+          ],
+          "x-ms-apimTemplateParameter.propertyParentPath": "@body()",
+          "x-ms-apimTemplateParameter.propertySubPath": "assignments",
+          "x-ms-apimTemplateParameter.keyWithinCollectionPath": "userId",
+          "x-ms-apimTemplateParameter.newPropertyPath": "@body().assignments",
+          "x-ms-apimTemplateParameter.retainKey": "false",
+          "x-ms-apimTemplate-policySection": "Request"
+        },
+        "templateId": "ConvertArrayToObject",
+        "title": "Convert array of assignees to object"
+      },
+      {
+        "parameters": {
+          "x-ms-apimTemplate-operationName": [
+            "GetTask_V2",
+            "CreateTask_V3",
+            "UpdateTask_V3",
+            "UnassignUsers",
+            "AssignUsers"
+          ],
+          "x-ms-apimTemplateParameter.propertyParentPath": "@body()",
+          "x-ms-apimTemplateParameter.propertySubPath": "assignments",
+          "x-ms-apimTemplateParameter.newPropertyPath": "@body()._assignments",
+          "x-ms-apimTemplateParameter.keyName": "userId",
+          "x-ms-apimTemplateParameter.valueName": "value",
+          "x-ms-apimTemplate-policySection": "Response"
+        },
+        "templateId": "ConvertObjectToArray",
+        "title": "Convert object of assigness to array for a task"
```

Comment:
Minor comment: Should this be `assignees` or `assignments`?

Reply:
Fixed. Thanks.